test/controllers/page_controller_test.exs

```diff
@@ -22,23 +22,35 @@ defmodule Pairmotron.PageControllerTest do
     setup do
       {:ok, usera} = Pairmotron.Repo.insert(%Pairmotron.User{name: "usera", email: "a", active: true})
       {:ok, userb} = Pairmotron.Repo.insert(%Pairmotron.User{name: "userb", email: "b", active: true})
-      conn = get build_conn, page_path(build_conn, :show, 2016, 40)
+      {year, week} = Timex.iso_week(Timex.today)
```

Comment:
Glad you like this, because I felt slightly dirty doing this. Not a huge issue, but this will fail at midnight on Sundays if you run the tests at just the right time lol.